main.nf

```diff
@@ -127,81 +176,115 @@ if(params.readPaths){
         .from(params.readPaths)
         .map { file(it) }
         .ifEmpty { exit 1, "params.readPaths was empty - no input files supplied" }
-        .into { raw_reads_fastqc; raw_reads_trimgalore }
+        .into { raw_reads_fastqc; raw_reads_trimgalore; raw_reads_mirtrace }
 } else {
     Channel
         .fromPath( params.reads )
         .ifEmpty { exit 1, "Cannot find any reads matching: ${params.reads}" }
-        .into { raw_reads_fastqc; raw_reads_trimgalore }
+        .into { raw_reads_fastqc; raw_reads_trimgalore; raw_reads_mirtrace }
 }
 
 
 // Header log info
-log.info """=======================================================
-                                          ,--./,-.
-          ___     __   __   __   ___     /,-._.--~\'
-    |\\ | |__  __ /  ` /  \\ |__) |__         }  {
-    | \\| |       \\__, \\__/ |  \\ |___     \\`-._,-`-,
-                                          `._,._,\'
-
-nf-core/smrnaseq : Small RNA-Seq Best Practice v${params.version}
-======================================================="""
+log.info nfcoreHeader()
+if(workflow.revision) summary['Pipeline Release'] = workflow.revision
```

Comment:
This line should be after the line below it? It was failing for me when I tested locally because `summary` hasnt been initialised yet.

Reply:
ok, I will fix that. i didn't see the warning for some reason.